### ISSUE TITLE
(question/feature request) Summing nodes that roll differently per seed

### ISSUE THREAD

**Issue by ryankennedy712** [KeshHere/POE-Timeless-Jewel-Finder#7]
is it possible to have this program search through a glorious vanity and add up all aura effect nodes? big and small? with this jewel "commanding presence" doesnt always roll the same value. think its like 5-10 or something.

**Comment by KeshHere** [comment]
Hey thanks for the interest, my current program only looks for notable passives, not the small nodes so I don't think it would be possible to do this.

In any case this has not been updated for 3.20, so I would suggest to use a web interface from vilsol. Google "Vilsol jewel github", it would be github.io link, it's pretty easy to use as well afaik.

**Comment by ryankennedy712** [comment]
yeah. vilsol's doesnt do what i wanted either. Thanks for the response!!! (vilsol hasnt yet. haha)
